README.md

```diff
@@ -1,12 +1,12 @@
-<h1 align="center">Exokit Studio</h1>
+<h1 align="center">Exoland</h1>
 <p align="center"><b>An interface for the <a href="https://github.com/exokitxr/exokit">Exokit Engine</a>.</b></p>
 
 <p align="center">
-  <a href="https://github.com/exokitxr/studio/releases"><img src="https://img.shields.io/github/downloads/exokitxr/studio/total.svg"></a>
-  <a href="https://www.npmjs.com/package/exokit-studio"><img src="https://img.shields.io/npm/v/exokit-studio.svg"></a>
-  <a href="https://github.com/exokitxr/studio/issues"><img src="https://img.shields.io/github/issues/exokitxr/studio.svg"></a>
-  <a href="https://github.com/exokitxr/studio"><img src="https://img.shields.io/github/forks/exokitxr/studio.svg"></a>
-  <a href="https://github.com/exokitxr/studio"><img src="https://img.shields.io/github/stars/exokitxr/studio.svg"></a>
+  <a href="https://github.com/exokitxr/exoland/releases"><img src="https://img.shields.io/github/downloads/exokitxr/exoland/total.svg"></a>
+  <a href="https://www.npmjs.com/package/exokit-exoland"><img src="https://img.shields.io/npm/v/exokit-exoland.svg"></a>
```

Comment:
will need to push to npm with name updated

Reply:
Published.